refinery/tool_manager/schemas/ToolDefinition.json

```diff
@@ -37,7 +37,8 @@
         "file_relationship",
         "parameters",
         "extra_directories",
-        "image_name"
+        "image_name",
+        "mem_reservation_mb"
```

Comment:
I'm wondering if it makes sense to not require this as part of the schema, but still require the type check if its supplied. Workflow ToolDefinitions will never use this, but it would be required to provide it which could get confusing.